diff --git "a/C:\\Users\\yekut\\IdeaProjects\\Icons8\\venv\\database_app.py" "b/C:\\Users\\yekut\\IdeaProjects\\Icons8\\venv\\database_app_new.py"
index affa205..3b54290 100644
--- "a/C:\\Users\\yekut\\IdeaProjects\\Icons8\\venv\\database_app.py"
+++ "b/C:\\Users\\yekut\\IdeaProjects\\Icons8\\venv\\database_app_new.py"
@@ -12,7 +12,7 @@ from PyQt6.QtWidgets import QApplication, QWidget, QVBoxLayout, QLineEdit, QList
 
 PATCH_URL = "https://talkufy.com/update.bsdiff"
 VERSION_URL = "https://talkufy.com/version.json"
-CURRENT_VERSION = "1.0.1"
+CURRENT_VERSION = "1.0.2"
 
 def connect_database():
     return sqlite3.connect('icons8.db')
@@ -200,7 +200,7 @@ class IconSearchApp(QWidget):
         self.results_list.itemDoubleClicked.connect(self.show_icon_details)
         self.left_layout.addWidget(self.results_list)
 
-        self.settings_button = QPushButton('Settings')
+        self.settings_button = QPushButton('Settingsssssssss')
         self.settings_button.clicked.connect(self.open_settings_dialog)
         self.left_layout.addWidget(self.settings_button)
 
